client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -49,14 +56,144 @@ object CapiModelEnrichment {
         isMedia -> Media,
         isReview -> Review,
         tagExistsWithId("tone/analysis") -> Analysis,
-        isComment -> Comment,
+        isCommentDesign -> Comment,
         tagExistsWithId("tone/features") -> Feature,
-        liveBlog -> Live,
-        deadBlog -> Article
+        isLiveBlog -> Live,
+        isDeadBlog -> Article
       )
 
-      val result = predicates.collectFirst { case (predicate, design) if predicate(content) => design }
+      val result = getFromPredicate(content, predicates)
       result.getOrElse(defaultDesignType)
     }
   }
+
+  implicit class RenderingFormat(val content: Content) extends AnyVal {
+
+
+    def design: Design = {
+
+      val defaultDesign: Design = ArticleDesign
+
+      val isPhotoEssay: ContentFilter = content => content.fields.flatMap(_.displayHint).contains("photoessay")
+
+      val predicates: List[(ContentFilter, Design)] = List(
+        tagExistsWithId("artanddesign/series/guardian-print-shop") -> PrintShopDesign,
+        isMedia -> MediaDesign,
+        isReview -> ReviewDesign,
+        tagExistsWithId("tone/analysis") -> AnalysisDesign,
+        isCommentDesign -> CommentDesign,
+        tagExistsWithId("tone/features") -> FeatureDesign,
+        tagExistsWithId("tone/recipes") -> RecipeDesign,
+        tagExistsWithId("tone/matchreports") -> MatchReportDesign,
+        tagExistsWithId("tone/interview") -> InterviewDesign,
+        tagExistsWithId("tone/editorials") -> EditorialDesign,
+        tagExistsWithId("tone/quizzes") -> QuizDesign,
```

Comment:
Are we also missing `Interactive` design? https://github.com/guardian/types/blob/main/src/format.ts#L32

Reply:
I don't think we technically support an `Interactive` design on DCR? It's not in our [`decideDesign`](https://github.com/guardian/dotcom-rendering/blob/main/src/web/lib/decideDesign.ts) function at least. If it does need supporting, I can add it in here and that way we'll just need to add it to the `decideDesign` function to translate it from the Scala JSON output to whatever Typescript needs.

Reply:
Should the be an `Interactive` `Design` type? I'm not sure what it represents? @JamieB-gu is there a need for this in Apps or could we remove it?

Reply:
It's in the [apps-rendering decision functions](https://github.com/guardian/apps-rendering/blob/1062b75faf0848cb6ae2bf4fd894b9a320352ce1/src/item.ts#L286-L290), so I'll add it in. If it's in here and then removed, it's not a difficult thing to rip out (which is part of the motivation for this work in the first place). 

I think that the work here should be done with a focus on targeting a specific version of `@guardian/types`, and at this point this is ever so slightly ahead of the current version by having different `Design`s for dead and live blogs, as well as removing the `Column` display type. But after this initial step of aligning the two, the goal should be to update the two in parallel, with references to the appropriate version of each within the changelog

Reply:
`Design.Interactive` is at the moment [derived from](https://github.com/guardian/apps-rendering/blob/c671b3e4d0a23a8adbe3ad93443076b0f7894fc7/src/capi.ts#L40) `ContentType.INTERACTIVE`. I believe it's there so that we can build the empty placeholder articles for full-page interactives (`ng-interactive`/`type/interactive` - example [here](https://www.theguardian.com/environment/ng-interactive/2020/oct/05/climate-data-dashboard-carbon-atmosphere-sea-level-arctic-ice)). Note that we explicitly render these [without any styles](https://github.com/guardian/apps-rendering/blob/986336e5970154754f4f9a294961a4e17569667b/src/components/body.tsx#L56) because the interactive code takes over.

I'm curious as to how DCR models these kinds of articles if it doesn't use the `Design` 🤔?